### DISCUSSION TITLE
Possible bug? fontbakery.com

### DISCUSSION THREAD

**Discussion by panosvassiliou** [fonttools/fontbakery#4487] [Q&A]
For the last couple of days the online version is on 'loading mode', it doesn't start. Is this a bug, is there something I should do to get it started?

![Screenshot 2024-02-05 at 10 05 32 AM](https://github.com/fonttools/fontbakery/assets/72914151/6f86923b-54ce-4811-82f6-04289edab18a)

**Comment by simoncozens** [comment]
Fixed it...
